script/deployPoolWithRenzoStabilityAndMintLiquidity.s.sol

```diff
@@ -34,9 +34,11 @@ contract CreatePoolAndAddLiquidityScript is Script, Constants, Config {
     // Hook configuration
     IRateProvider rateProvider =
         IRateProvider(0xDb6df3559D2d96985062F0824442550CA7715960);
-    uint24 minFee = 100;
-    uint24 maxFee = 10_000;
-    address ezETH = 0x2416092f143378750bb29b79eD961ab195CcEea5;
+    uint24 minDynamicFee = 2_500;
+    uint24 maxDynamicFee = 10_000;
+    uint24 defaultFee = 100;
```

Comment:
if the objective is to vampire  the vanilla 1 bip, i wonder if the default fee should be something closer to 0.95 bips 🤔 

hard to say what the magic number is, but yeah just some food for thought

Reply:
Yeah I think we can start with a lower default fee on custom pool to drive volume from vanilla pool to custom fee hook pool.